nautobot/docs/release-notes/version-2.0.md

```diff
@@ -18,6 +18,26 @@ Added Site Model Fields to Location. Location Model now has `asn`, `comments`, `
+
+4. If you have both `Site` and `Region` instances in your existing database, a `Region` LocationType will be created and for each legacy `Region` instance, a corresponding `Location` instance with the same attributes (`name`, `description` and etc) will be created. A `Site` LocationType will be created with the new `Region` LocationType as the parent and every root level `LocationType` in your database will have the new `Site` LocationType as a parent. For each legacy `Site` instance, a corresponding `Location` instance with the same attributes (`name`, `description`, `tenant`, `facility`, `asn`, `latitude`, `longitude` and etc) will be created. If you have `Site` instances in your database without a `Region` assigned to them, one additional location named "Global Region" of LocationType `Region` will be created and all the locations of LocationType `Site` created from the legacy region-less `Site` instances will have "Global Region" as their parent. In addition to that, models (`CircuitTermination`, `Device`, `PowerPanel`, `RackGroup`, `Rack`, `Prefix`, `VLANGroup`, `VLAN`, `Cluster`) with an old `Site` instance assigned but without its `location` attribute specified will have its `location` point to the new `Location` of `Site` LocationType. These Models with `location` attribute specified will remain unchanged.
+This scenario is very similar to case 3 above in which you only have Site instances in your existing database. In this scenario, however, you also have Region instances.
+
+    For Example:
+        There are two `Site` instances and one `Region` instance in your existing database:
+        The `Region` with name **America** has one child `Site` instance named **AMS01**. And the other `Site` instance named **AUS01** is not associated with any regions (`region` attribute is set to `None`). Site **AMS01** is the base site for two top level locations with names **root-01** and **root-02** respectively in the your existing database. During the data migration, `Region` LocationType and a location of this LocationType named **America** with all the information will be created. `Site` LocationType with its `parent` set as the new LocationType `Region` and a location of `Site` LocationType named **AMS01** with all the information (`asn`, `latitude`, etc) will be created. Location **AMS01** will have Location **America** as its `parent` and locations **root-01** and **root-02** will have location **AMS01** as their `parent`. As for `Site` instance **AUS01**, since it does not have a `Region` instance associated with it, its corresponding location **AUS01** will have a new location named **Global Region** of LocationType `Region` as its parent.
```

Comment:
```suggestion
        There are two `Site` instances and one `Region` instance in your existing database. The `Region` with name **America** has one child `Site` instance named **AMS01**. And the other `Site` instance named **AUS01** is not associated with any `Region` (`region` attribute is set to `None`). The `Site` **AMS01** is the base `Site` for two top-level `Location` objects with names **root-01** and **root-02** respectively. During the data migration, a `LocationType` named **Region** and a `Location` of this `LocationType` named **America** with all the same information will be created.  The `LocationType` named **Site** with its `parent` set as the new `LocationType` **Region** and a `Location` of `LocationType` named **AMS01** with all the same information (`asn`, `latitude`, etc.) will be created. The `Location` **AMS01** will have `Location` **America** as its `parent` and each `Location` **root-01** and **root-02** will have `Location` **AMS01** as its `parent`. Finally, the `Site` instance **AUS01**, since it does not have a `Region` instance associated with it, its corresponding `Location` **AUS01** will have a new `Location` named **Global Region** of `LocationType` **Region** as its `parent`.
```

Reply:
Consider splitting this into a list of data/actions as well.